src/style.css

```diff
@@ -1,3 +1,95 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should provide a generic font family as a fallback. For example, 'Roboto', sans-serif could be followed by a generic family like 'Arial', sans-serif.